Driverless AI/natural-language-processing-tutorial-sentiment-analysis /.ipynb_checkpoints/natural-language-processing-tutorial-sentiment-analysis-checkpoint.md

```diff
@@ -611,60 +604,49 @@ This data comes from  [Crowdflower's Data for Everyone library](http://www.crowd
 
 ### Steps:
 
-1\. Import the dataset from here:
+1. Import the dataset from here:
 
 -   [Airline-Sentiment-2-w-AA](http://airline-sentiment-2-w-aa/)
 
 Here are some samples from the dataset:
 
-![Challenge_dataset](assets/Challenge_dataset.png)
+![](assets/Challenge/dataset.png)
 
-2\. Split the dataset into a training set and a testing set in 80:20 ratio.
+2. Split the dataset into a training set and a testing set in 80:20 ratio.
```

Comment:
@Castro38 rewrite this: "Split the dataset into a training set and a testing set in 80:20 ratio."

to this: "Split the dataset into a training set and a testing set in an 80:20 ratio."

Reply:
@5675sp Done